screens/courses/ViewCourseScreen.js

```diff
@@ -23,11 +26,26 @@ class ViewCourseScreen extends React.Component {
       sessions: [],
       course : { },
       isLoading : true,
+      students: { },
     }
   }
 
+  static navigationOptions = ({ navigation }) => {
+    const { params = {} } = navigation.state;
+    return {
+        headerRight: <Button title="Enroll Student" onPress={() => params.handleCreate()} />
```

Comment:
again if you have any performance issues look into refactoring this since you have a ton of arrow functions in render